python/cuml/neighbors/kneighbors_classifier.pyx

```diff
@@ -317,6 +319,37 @@ class KNeighborsClassifier(NearestNeighbors, ClassifierMixin):
         return final_classes[0] \
             if len(final_classes) == 1 else tuple(final_classes)
 
+    def score(X, y, sample_weight=None, convert_dtype=True):
```

Comment:
Just taking a quick peek, would it be better for KNeighbors classifer to inherit from the ClassifierMixin instead of defining `score` here explicitly?

Reply:
I think this should definitely extend the ClassifierMixin. I'm assuming the classifier mixin doesn't account for multiple outputs though, does it? We might need to make the small modification to have it compute the accuracy for each potential column in the labels array and average them. I think this will be easy to do generally, though.

Reply:
Was curious why the `KneighborsClassifier` in Scikit-learn extends the `ClassifierMixin` but the `accuracy_score` doesn't support multiple outputs. It throws an error.